app/static/js/slcNewProposal.js

```diff
@@ -72,7 +72,7 @@ function validateForm() {
   if (valid) {
     $(".step")[currentTab].className += " finish"
   }
-  return valid;
+  return true;

```

Comment:
Are we wanting our validation to always be true?

Reply:
Hila had made this function to validate the proposal form however she mentioned that it would be more effective if we created a more general validation and I assumed that would fall under a separate issue.

Reply:
I would leave the specific one if it works. We can replace it once the general validation exists.